src/components/About.js

```diff
@@ -1,7 +1,149 @@
+/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 import React from 'react';
+import { AboutText } from './Text';
+import cute from '../img/cute.png';
+
+// NOTE: don't need justify content in css b/c it is always centered??? << this may be a sike nah!!!
```

Comment:
Won't be always justified, it's probably just showing up as justified because of a higher level css format but you would use something like `justify-content` under flexbox to center: https://css-tricks.com/almanac/properties/j/justify-content/

Reply:
lol thanks appreciate the clarification on that one